lib/core.js

```diff
@@ -652,57 +654,48 @@ AT.prototype.setupRoutes = function() {
         var path = options.path; // Default provided...
         var template = options.template || "fullPageAtForm";
         var layoutTemplate = options.layoutTemplate || defaultLayout;
+        var renderLayout = options.renderLayout || {};
+        var contentRange = options.contentRange;
 
         // Possibly adds token parameter
         if (_.contains(["enrollAccount", "resetPwd", "verifyEmail"], route)){
             path += "/:paramToken";
             if (route === "verifyEmail")
-                Router.route(path, {
-                    name: name,
-                    template: template,
-                    layoutTemplate: layoutTemplate,
-                    onRun: function() {
+                FlowRouter.route(path, {
+                    middlewares: [function(path, next) {
                         AccountsTemplates.setState(route);
                         AccountsTemplates.setDisabled(true);
-                        var token = this.params.paramToken;
+
+                        next();
+                    }],
+                    action: function(params) {
+                        renderLayout[contentRange] = template;
+                        FlowLayout.render(layoutTemplate, renderLayout);
```

Comment:
renderLayout needs to be defined per route like so:
````javascript
renderLayout: {
    nav: "nav",
    footer: "footer"
}
````
and is {} if there is nothing defined.